feat: upgrade dependencies

WietekeNijmegen opened the pull request:
Fixes #
Reverted sample files
Pipeline finally succeeded
Getest op acc, gegevens ingeladen

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13125667574

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #887 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13147541065

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #888 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13169215374

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #889 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13190567584

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #890 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13210334139

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #891 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13220638977

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #892 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13231115472

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #893 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13253261634

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #894 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13275274605

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #895 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13297680129

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #896 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13319817558

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #897 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13339823523

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #898 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13350006588

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #899 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13360346390

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #900 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13380747965

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #901 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13402813451

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #902 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13425135407

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #903 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13447368512

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #906 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13467585026

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #907 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13477950005

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #908 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13488434934

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #909 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13510821343

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #910 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13533774068

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #911 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13556150244

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #912 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13578633323

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #913 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13598944175

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #914 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13610314984

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #915 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13643418607

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #916 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13665731087

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #917 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13688399578

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #918 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13711025033

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #919 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13731771445

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #920 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13742914363

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #921 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13754119222

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #922 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13777614792

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #923 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13800956154

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #924 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13824263925

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #925 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13847270011

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #926 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13867573537

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #927 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13878100282

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #928 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13912654320

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #929 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13936163647

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #930 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13959368583

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #931 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/13982024097

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #932 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14003262534

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #933 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14013790052

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #934 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14024627010

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #935 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14048833553

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #936 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14072790759

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #937 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14096167154

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #938 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14119341416

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #939 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14140258879

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #940 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14150871235

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #941 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14161572153

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #942 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14184343165

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #943 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14208826124

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #945 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14232285808

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #946 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14255411441

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #947 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

WietekeNijmegen committed:
Merge branch 'production' into acceptance

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

WietekeNijmegen committed:
fix: remove sample files

WietekeNijmegen committed:
fix: sample files added statics again

WietekeNijmegen committed:
fix: upgrade dependencies

WietekeNijmegen committed:
fix: statics imports

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing commented on src/statics.ts:
Zijn de projen lint instellingen veranderd?

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14276451140

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #949 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/14287249555

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #950 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
chore: trigger builds
